build.sbt

```diff
@@ -16,6 +16,8 @@ ThisBuild / tlSonatypeUseLegacyHost := true
 
 ThisBuild / crossScalaVersions := Seq(scala212, scala213, scala3)
 ThisBuild / scalaVersion := scala213 // the default Scala
+ThisBuild / githubWorkflowJavaVersions += JavaSpec.temurin("11")
```

Comment:
Probably, you want to `:= Seq(JavaSpec.temurin("8"))`. Since this will _add_ a JDK 11 job, instead of replacing the JDK 8 job.